src/components/MessageList/MessageList.js

```diff
@@ -71,6 +72,10 @@ const MessageList = ({ messages, handleGoBack }) => {
     }
   };
 
```

Comment:
Just one suggestion, we could add a toast notification that "Message deleted successfully" and handle any errors there.